data/jayk-gupta.json

```diff
@@ -0,0 +1,27 @@
+{
+  "name": "Jay kumar Gupta",
+  "displayStatsPublic": true,
+  "type": "personal",
+  "bio": "CSE Undergraduate,passionate about frontend developement",
+  "avatar": "https://github.com/jayk-gupta.png",
+  "links":[
+    {
+    "name": "Follow me on Twitter",
+      "url":"https://twitter.com/jayk_gupta",
+      "icon":"FaTwitter"
+      
+    },
+    {
+       "name": "Follow me on LinkedIn",
+      "url":"https://www.linkedin.com/in/jay-kumar-gupta-ba0434201/",
+      "icon":"FaLinkedIn"
```

Comment:
use this icon for LinkedIn `FaLinkedin`

Reply:
We have both icons: so no worries. 
<img width="157" alt="Screenshot 2023-01-12 at 7 30 41 PM" src="https://user-images.githubusercontent.com/51878265/212086497-09ba2c94-401c-4dd6-a870-f9058ff739ea.png">


Also, we do auto-formatting by GitHub Actions after merge